test_integration/erc20_gateway/01_setup/01_setup.ts

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+
+import shared, { ContractEntity } from '../shared';
+import Utils from '../Utils';
+import { ERC20Gateway } from '../../../interacts/ERC20Gateway';
+import { ERC20Cogateway } from '../../../interacts/ERC20Cogateway';
+import Assert from '../Assert';
+
+describe('Contract Setup', async (): Promise<void> => {
+
+  let ERC20Gateway: ContractEntity<ERC20Gateway>;
+  let ERC20Cogateway: ContractEntity<ERC20Cogateway>;
+
+  before(async() => {
+    ERC20Gateway = shared.contracts.ERC20Gateway;
+    ERC20Cogateway = shared.contracts.ERC20Cogateway;
+
+    shared.metachainId = shared.web3.utils.randomHex(32);
+    const utilityTokenMasterCopy = await shared.artifacts.UtilityToken.new();
```

Comment:
Please move deployment to deployment file.

Reply:
Done